src/Docker.DotNet/Endpoints/MiscellaneousOperations.cs

```diff
@@ -4,6 +4,7 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Docker.DotNet.Models;
+using Newtonsoft.Json;
```

Comment:
Same. If you remove the JsonConvert reference and use `this._client.JsonSerializer` we can remove this too